frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -40,6 +40,12 @@ end
 function ReaderWikipedia:registerKeyEvents()
     if Device:hasKeyboard() then
         self.key_events.ShowWikipediaLookup = { { "Alt", "W" }, { "Ctrl", "W" } }
+        if Device.model == "Kindle3" then
+            -- Override previous shortcut on kindle3 as the kernel designates Alt+W as "2".
+            -- Any user-facing documentation should _still_ advertise this shortcut as Alt+W, as
+            -- that is what the user must input in order to use it.
+            self.key_events.ShowWikipediaLookup = { { Device.AltPlusQWER["W"] } }
+        end
```

Comment:
best practices question, should I keep this or be a reckless boy and use 

```lua
self.key_events.ShowWikipediaLookup = { { "Alt", "W" }, { "Ctrl", "W" }, { Device.AltPlusQWER["W"] } }
```

Reply:
Keep it as it was (with `if Device.AltPlusQWER then` instead of `if Device.model == "Kindle3" then`
Your suggestion, if used without the `if Device.model == "Kindle3" then` would crash on devices that don't have a AltPlusQWER.
But If used inside the `if`, it lies, as `{ "Alt", "W" }` can't happen if it is mapped by the kernel to "2".

Reply:
> would crash on devices that don't have a AltPlusQWER.

oh yeah. oh well, good thing I asked.